src/envo/envo-edit.owl

```diff
@@ -39106,6 +39194,75 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510041>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03510041> "wood stain"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03510041> <http://purl.obolibrary.org/obo/ENVO_03510040>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520000> (low atmospheric level)
```

Comment:
Sounds good - but a little odd not to have a definition. I understand the reason behind it though, especially given the WMO approach (see editor note)

Reply:
I've added a link to the German wikipedia page as an annotation to the editor note. On there, layers which coincide with the extent of the low étage are mentioned. However, I am hesitant to add them to help this definition because I haven't yet seen them mentioned elsewhere.